docs/registries.conf.5.md

```diff
@@ -0,0 +1,28 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is YAML.
+
+# FORMAT
+The [YAML_format][yaml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`.
+
+The following example configuration defines two registries and one insecure registries.
+
+```
+registries:
+    - registry1.com
+    - registry2.com
+
+insecure_registries:
+    - registry3.com
+```
```

Comment:
@runcom showed a registry in his blog that lives at:  /etc/crio/crio.conf and has the format of 
registries = [
“registry.example.xyz”,
]

Is it possible for us to use one common file?
https://docs.google.com/document/d/1cBT5qPxK-ooSfGsCYan7hRAzIlhZP-JFrvvohWQcwk4/edit